Text-Based-Adventure/Text-Based-Adventure/main.swift

```diff
@@ -0,0 +1,459 @@
+//
+//  main.swift
+//  Text-Based-Adventure
+//
+//  Created by Kelby Mittan on 9/30/19.
+//  Copyright © 2019 Kelby Mittan. All rights reserved.
+//
+
+import Foundation
+
+
+print("Enter your name!!")
+let enterName = readLine()
+
+print()
+
+print("Hello \(enterName ?? "playa"), Welcome to Ascend NYC..... on a scale of \"1-10\" how adventurous are you feeling about treking to the top of a New York City landmark today?")
+
+let howAdventurous = readLine()
+
+let howAdenturousAsAnInt = Int(howAdventurous ?? "") ?? 7
+
+//var isPlaying = false
+
+var trophy = (landmark: String(), level: String())
+
+//if isPlaying == false {
```

Comment:
Heads up, best practice is to remove commented out code before submitting, something to keep in mind for future projects.